.changeset/blue-sloths-repair.md

```diff
@@ -0,0 +1,12 @@
+---
+'@commercetools-test-data/standalone-price': minor
+'@commercetools-test-data/customer-group': minor
+'@commercetools-test-data/product-type': minor
+'@commercetools-test-data/channel': minor
+'@commercetools-test-data/commons': minor
+'@commercetools-test-data/core': minor
+---
+
+The main change is about the `core` package where we are introducing support for writing test data models using new implementation patterns which makes the process simpler. Also, the resulting code will be more maintainable.
+
+You can head over [here]() for updated documentation about those new patterns.
```

Comment:
I'll update this link once the [docs PR](https://github.com/commercetools/test-data/pull/687) is merged.